docs/percona-monitoring-and-management-doc-template.md

```diff
@@ -0,0 +1,66 @@
+
+# Feature Name
+
+!!! caution alert alert-warning "Important/Caution"
+    Crucial points that need emphasis:
+
+    - Important: A significant point that deserves emphasis.
+    - Caution: Used to mean 'Continue with care'.
+
```

Comment:
just to confirm - we're not making these definitions visible for users, right?

Reply:
Yes. These are just placeholders.